### DISCUSSION TITLE
24-9-9 六分街/基沃托斯/提瓦特/......每日音乐推荐（1）Beyond乐队——Good Time

### DISCUSSION THREAD

**Discussion by sensei-is-playing-GenShin** [share121/inter-knot#4653] [其他]
Good Time是Beyond乐队于1999年发行专辑《Good Time》里的一首经典曲目（虽然说是经典，但还是在大多数人看来还是比不过如《喜欢你》《光辉岁月》《真的爱你》等商业化的歌曲）由黄贯中作词作曲，是一首典型的流行摇滚。同时这张专辑也是Beyond乐队（后来成员单飞歌不算）最后的一张专辑（应该是吧，反正我看03年重组后也没有出过新专辑了），Beyond乐队也在专辑发行后的1999年年底宣布解散。
![微信截图_20240909001302](https://github.com/user-attachments/assets/c455abc4-9e06-44f3-a1d1-83a16ecbc1ee)

这是这个系列的第一章，以后一定一定会日更的。当然了，如果文章中有错误的地方欢迎在评论区里提出，如果有什么好歌也可以在评论区留言，有时间我都会一一回复的，谢谢大家。

**Comment by Stormlocken** [comment]
顶

**Reply by sensei-is-playing-GenShin** [reply]
谢

**Comment by sensei-is-playing-GenShin** [comment]
音频网盘链接：https://pan.baidu.com/s/1Uj_pQwqOKrRPyTZ-IAts8A?pwd=kbmp  提取码: kbmp

**Reply by sensei-is-playing-GenShin** [reply]
有多种音质，可以根据MP3或手机的内存自主选择
